valerymo/integreatly-operator
# MGDAPI-2681 Showing dashboards in OO grafana

# Issue link

https://issues.redhat.com/browse/MGDAPI-2681

# What

Moving the dashboards from the old monitoring stack to the new.

# Verification steps




- Install this branch
- Log into the Grafana found in the `redhat-rhoam-observability` namespace. Get route `oc get routes -n redhat-rhoam-observability --no-headers | grep grafana | awk '{print $2}' | xargs -I {} echo https://{}`
- Check that the dashboards are all in grafana. 
  - current problem is the sso dashboards are not being picked up
  - Data will not be showing in the dashboards. This is should be fixed in a different ticket

Expected Dashboards
- Critical SLO summary
- CRO Resources
- Endpoints Detailed
- Endpoints Report
- Endpoints Summary
- Resource Usage By Namespace
- Resource Usage for Cluster
- SLO SSO Availability - 5 xx HAProxy Errors

Currently Missing Dashboards
- Keycloak Metrics (Folders: redhat-rhoam-rhsso,  redhat-rhoam-user-sso)
